lib/search/global/projects.js

```diff
@@ -33,8 +39,7 @@ module.exports = client => async (term = '', query = {}) => {
   const fields = [
     'title',
     'licenceHolder.lastName',
```

Comment:
There was chat about adding first name as well, no?

Reply:
Indeed, was going to fix that in a separate PR, but can do it here if you prefer?

Reply:
Done.